it/pages/TwoEyePageReviewControllerISpec.scala

```diff
@@ -73,8 +73,8 @@ class TwoEyePageReviewControllerISpec extends IntegrationSpec {
 
           ExternalGuidanceStub.approval2iPageReviewComplete(Status.NO_CONTENT, Json.parse("{}"))
 
-          val request: WSRequest = buildRequest("/2i-page-review/oct90005/pageUrl")
-          val response: WSResponse = await(request.post(Json.obj("answer" -> YesNoAnswer.Yes.toString)))
+          val request: WSRequest = buildRequest("/2i-page-review/oct90005/pageUrl?title=Title")
+          val response: WSResponse = await(request.post(Json.obj("answer" -> YesNoAnswer.Yes.toString, "title" -> "Title")))
```

Comment:
As with FactCheckPageReviewControllerISpec

Reply:
done

Reply:
This causes the test to failed, removed